src/devices/src/virtio/net/device.rs

```diff
@@ -684,6 +684,11 @@ impl VirtioDevice for Net {
         // TODO: to be removed
         Ok(())
     }
+
+    #[cfg(test)]
+    fn set_avail_features(&mut self, avail_features: u64) {
```

Comment:
Are there any plans to use this to test net, block and vsock device functionality as well? If not, should we consider implementing this as part of the DummyDevice instead of adding it to the `VirtioDevice` trait?

Reply:
Done. 👍